docs/eventing/channels/channels-crds.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Available Channels CRDs"
```

Comment:
maybe call it Available Channels as this has both the CCP and the CRDs?
Might be worth renaming the file also to channels.md ??

Reply:
reduced to generic "Channels"